dist/stampit.js

```diff
@@ -710,9 +738,13 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
```

Comment:
Yeah. I noted. My comments above are related to your sample code.

Check it yourself :)